test/Sentry.Serilog.Tests/Sentry.Serilog.Tests.csproj

```
@@ -4,8 +4,18 @@
     <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Primitives" Version="2.1.1" />
+    <PackageReference Include="Serilog" Version="2.7.1" />
+    <PackageReference Include="Serilog.Settings.Configuration" Version="3.1.0" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="../../src/Sentry.Serilog/Sentry.Serilog.csproj" />
+    <ProjectReference Include="..\Sentry.Testing\Sentry.Testing.csproj" />
```

Just for consistency (see line above):
```suggestion
    <ProjectReference Include="../Sentry.Testing/Sentry.Testing.csproj" />
```

---

Good eye. This one was odd--visual studio added that reference and yet it was showing up with the yellow triangle/couldn't find it in the solution explorer.